spec/models/amusement_park_spec.rb

```diff
@@ -4,4 +4,9 @@
   describe "relationships" do
     it { should have_many(:rides) }
   end
+
+  describe "validations" do
+    it {should validate_presence_of(:name)}
+    it {should validate_presence_of(:admission_cost)}
+  end
```

Comment:
You created the method `unique_mechanics` but did not test it here. Make sure to test all of your model methods as well.